draft-ietf-tls-dtls-connection-id.md

```diff
@@ -345,6 +345,33 @@ data the following modification is made to the additional data calculation.
                       length_of_DTLSInnerPlaintext;
 ~~~
 
+# Peer Address Update {#peer-address-update}
+
+When a datagram with a CID is received that has a source address different than the one currently associated with the DTLS connection, the receiver MUST NOT replace the source address currently stored in the association with the source address specified in the received datagram unless the following conditions are met:
+
+- The received datagram has been cryptographically verified using the DTLS record layer processing procedures.
+
+- The received datagram is "newer" (in terms of their epoch and sequence
+number) than the last datagram that successfully updated the source address.
+This condition ensures that replayed datagrams cannot be used to perform address 
+updates. Note that this condition does not help when the attacker is able to 
+replay datagrams that arrive earlier at the DTLS peer than the original datagram.
+
+- There is a strategy for performing a perform some reachability test. 
+No such test is defined in this	specification; {{!I-D.tschofenig-tls-dtls-rrc}} 
+defines an extension for such a DTLS protocol extension.
+
+The above ensures correctness of the protocol in presence of packet
```

Comment:
There is no "ensures" here.  And, as mentioned, I think that the reordering thing is an optimization only, the reordering point can be separated.

"The above is necessary to protect against attacks that use datagrams with spoofed addresses or replayed datagrams to trigger attacks."

Reply:
Good text for me. I included it.